contrib/libdbusmenu/template.py

```diff
@@ -0,0 +1,56 @@
+pkgname = "libdbusmenu"
+pkgver = "16.04.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-dumper",
+    "--disable-tests",
+    "--disable-static",
+    "--enable-gtk",
+]
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
+    "automake",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "atkmm-devel",
+    "dbus-glib-devel",
+    "gettext-devel",
+    "gtk-doc-tools",
+    "gtk+3-devel",
+    "glib-devel",
+    "gobject-introspection",
+    "json-glib-devel",
+    "vala-devel",
+]
+pkgdesc = "Library for passing menus over DBus"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "GPL-3.0-or-later AND LGPL-3.0-or-later AND LGPL-2.1-or-later"
+url = "https://launchpad.net/libdbusmenu"
+source = f"{url}/16.04/{pkgver}/+download/libdbusmenu-{pkgver}.tar.gz"
+sha256 = "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+# Glib tests fails
```

Comment:
the failures are because you're missing `checkdepends = ["dbus", "bash"]`

after that it still fails, because of unpackaged dbus-test-runner (https://launchpad.net/dbus-test-runner)

Reply:
Yep, the problem was because of missing bash